src/crate/client/connection.py

```diff
@@ -123,12 +130,16 @@ def __init__(self,
         self.lowest_server_version = self._lowest_server_version()
         self._closed = False
 
-    def cursor(self):
+    def cursor(self, cursor=None, **kwargs) -> Cursor:
```

Comment:
What's the `cursor=None` for?

Reply:
It probably slipped in from the role model implementation of PyAthena. I think it can be removed. Thanks for spotting it.

```
def cursor(self, cursor: Optional[Type[BaseCursor]] = None, **kwargs) -> BaseCursor:
```

-- https://github.com/laughingman7743/PyAthena/blob/2f2c0e11487426bd382f480a8468e9bc78c42156/pyathena/connection.py#L231